### PR TITLE
Register EXT_texture_compression_bptc|rgtc for ES

### PR DESCRIPTION
**Pull request by dgkoch**

### CODE REVIEW

**Comment by oddhack** [comment]
This needs conflicts resolved (every time I accept a PR that modifies the generated headers, it causes a conflict with pending PRs that modify the headers, alas).

**Comment by dgkoch** [comment]
@oddhack rebased

File: extensions/EXT/EXT_texture_compression_rgtc.txt [hunk 1]
```diff
@@ -226,28 +253,71 @@ Dependencies on OpenGL 2.0 or ARB_texture_non_power_of_two
...
+    COMPRESSED_SIGNED_RED_RGTC1_EXT         RED         SIGNED_BYTE
+    COMPRESSED_RED_GREEN_RGTC2_EXT          RG          UNSIGNED_BYTE
+    COMPRESSED_SIGNED_RED_GREEN_RGTC2_EXT   RG          SIGNED_BYTE
+
 Errors
 
     INVALID_ENUM is generated by CompressedTexImage1D
     or CompressedTexImage3D if <internalformat> is
-    COMPRESSED_LUMINANCE_LACT1_EXT, COMPRESSED_SIGNED_RED_RGTC1_EXT,
+    COMPRESSED_RED_RGTC1_EXT, COMPRESSED_SIGNED_RED_RGTC1_EXT,
```
Comment (oddhack) [inline extensions/EXT/EXT_texture_compression_rgtc.txt]: Parenthetically, I note that in addition to fixing the token (which should IMO be added to the change log), the *spelling* of the old COMPRESSED_LUMINANCE_LACT1_EXT token appears wrong in both EXT_texture_compression_{latc,rgtc}. Presumably that should be fixed as well?

Comment (dgkoch) [inline extensions/EXT/EXT_texture_compression_rgtc.txt]: Added note to changelog and fixed LACT typos in latc spec.

**Pull request merged**